### PR TITLE
es6-destructuring - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
Screenshot 2023-03-13 at 2 10 52 PM

### CODE REVIEW

File: es6-destructuring/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  <body>
+    <script>
+      const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353
+};
+
+      const { title, author, libraryID } = book1;
+      console.log('The title of the book is,', title, 'the author is ' + author + ', and the library id is', libraryID);
```
Comment (RobertGardner) [inline es6-destructuring/index.html:15]: Use interpolation.

File: es6-destructuring/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+      const book2 = {
+  title2: 'Les Fleurs du mal',
```
Comment (RobertGardner) [inline es6-destructuring/index.html:18]: The field names should be the same as in `book1` but when you destructure, put them into variables with a different name.

**Review by RobertGardner — CHANGES_REQUESTED** [review]

File: es6-destructuring/index.html (hunk 1 above)
Comment (Lambpato) [reply es6-destructuring/index.html:15]: i thought we shouldnt do that when console logging

File: es6-destructuring/index.html [hunk 3]
```diff
@@ -12,16 +12,16 @@
 };
 
       const { title, author, libraryID } = book1;
-      console.log('The title of the book is,', title, 'the author is ' + author + ', and the library id is', libraryID);
+      console.log(`The title of the book is, ${title}, the author is ${author}, and the library id is, ${libraryID}`);
```
Comment (RobertGardner) [inline es6-destructuring/index.html:15]: I just noticed that you used `,` in some of the places in your prior `console.log`. I keyed in on the `+`s. It's certainly legitimate to do it all with `,`s.

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**